src/allmydata/test/test_log.py

```diff
@@ -154,3 +154,17 @@ class LoggingObject1(tahoe_log.PrefixingLogMixin):
         obj.log("four")
         self.assertEqual([m[2] for m in self.messages],
                          ["grand", "par1", "par2", "msg1", "msg1"])
+
+    def test_native_string_keys(self):
+        """Keyword argument keys are all native strings."""
+        class LoggingObject17(tahoe_log.PrefixingLogMixin):
```

Comment:
Just a throw-away comment - love this naming convention.

Reply:
There's some hidden state involved, you can't just reuse the same class.